internal/cache/file/downloader/parallel_downloads_job.go

```diff
@@ -62,10 +62,8 @@ func (job *Job) downloadRange(ctx context.Context, dstWriter io.Writer, start, e
 
 	monitor.CaptureGCSReadMetrics(ctx, util.Parallel, end-start)
 
-	// Use of memory aligned buffer is not required if use of O_DIRECT is disabled.
+	// Use memory aligned buffer if O_DIRECT is enabled.
 	if job.fileCacheConfig.EnableODirect {
```

Comment:
It is recommended to put smaller code in if-block whenever there is else case. so its easier to understand the else part also.